initial-setup-gui.sh

```diff
@@ -22,6 +22,26 @@ TESTTYPE="initial-setup"
 . ${KSTESTDIR}/functions.sh
 . ${KSTESTDIR}/validate-lib-initial-setup.sh
 
+prepare() {
+    ks=$1
+    tmpdir=$2
+    export_additional_repo $tmpdir
```

Comment:
You can probably call this in the `kernel_args` function. The `prepare` function is basically for preparing a kickstart.

Reply:
Good point! :) That simplifies things quite a bit - thanks! :)